bin/eds-import-from-figma.js

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env node
+(async function () {
+  const fs = require('fs');
+  const jsonfile = require('jsonfile');
+  const yargs = require('yargs/yargs');
+
+  // eslint-disable-next-line import/extensions
+  const { hideBin } = require('yargs/helpers');
+  const { prompt } = require('enquirer');
+  const at = require('lodash/at');
+  const set = require('lodash/set');
+
+  const { default: ora } = await import('ora');
```

Comment:
Correct. This CLI tool's latest version is just ESM, and it was a hassle to try and move the whole thing over. I'm tempted to use the last version that supports `require` to avoid this one awkward line until this settles down